MC/bin/o2dpg_sim_workflow_anchored.py

```diff
@@ -258,21 +258,45 @@ def retrieve_MinBias_CTPScaler_Rate(ctpscaler, finaltime, trig_eff_arg, NBunches
     and calculates the interation rate to be applied in Monte Carlo digitizers.
     Uses trig_eff_arg when positive, otherwise calculates the effTrigger.
     """
+    trigger_effs = {
+        "pp": {
+            "1000": 0.68,
+            "6000": 0.737,
+            "default": 0.759
+        },
+        "pO": {
+            "default": 0.8222
+        },
+        "Op": {
+            "default": 0.8222
+        },
+        "OO": {
+            "default": 0.8677
+        },
+        "NeNe": {
+            "default": 0.9147
```

Comment:
@shahor02 @chiarazampolli : Hardcoding and maintaining these values in our script seems error-prone. Could we not have the trigger effs and related numbers stored in some GRP like CCDB object?